chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -327,6 +328,16 @@ void elemhideBlockedResource(
       : webContents->GetMainFrame());
 
     if (frameHost) {
+      if (is_subframe) {
+        LOG(WARNING) << "Adblock: moving to parent RenderFrameHost* for subframe";
+        frameHost = frameHost->GetParent();
```

Comment:
We don't have test pages to test for your case.
In case of 50statesclassifieds.com where i tested frame_id is not `0` so it's not main frame and `GetMainFrame()` is not called - it's some specific frame.

Probably for main frame case we shouldn't go to super frame..

Reply:
So, `frameHost` is not a hosting frame, it's basically a frame corresponding to `frameTreeNodeId`. So, the everything looks good.